### PR TITLE
[TASK] Use the new location of the database schema file

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ named `phplist`, and the password is `batterystaple`, the command looks like
 this:
 
 ```bash
-mysql -u phplist_test --password=batterystaple phplist_test < vendor/phplist/core/Database/Schema.sql
+mysql -u phplist_test --password=batterystaple phplist_test < vendor/phplist/core/resources/resources/resources/Database/Schema.sql
```
Comment (samtuke) [inline .github/CONTRIBUTING.md]: `/core/resources/resources/resources/Database/...` doesn't look right -- is it? Surely `/core/resources/Database/` ?

Comment (oliverklee) [reply .github/CONTRIBUTING.md]: Good catch! Fixed.

**Comment by oliverklee** [comment]
@samtuke Could you please re-review? Thanks!

**Comment by oliverklee** [comment]
@samtuke ping

**Comment by oliverklee** [comment]
Merging without a positive review so we can get this blocker out of the way.

**Comment by samtuke** [comment]
Thanks; sorry for the delay.

**Pull request merged**